src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+function convertToNumber(currency) {
+  return Number(currency.split(',').join('.').slice(1));
+}
+
+thead.addEventListener('click', (e) => {
+  const title = e.target.textContent;
+  const index = e.target.cellIndex;
+
+  const rows = [...tbody.rows];
+  const enlargedRows = rows.map((row) => [row.cells[index].textContent, row]);
+
+  enlargedRows.sort((row1, row2) => {
+    switch (title) {
+      case 'Name':
+      case 'Position':
+        return row1[0].localeCompare(row2[0]);
+
+      case 'Age':
+        return row1[0] - row2[0];
+
+      case 'Salary':
+        return convertToNumber(row1[0]) - convertToNumber(row2[0]);
```

Comment:
This is a code quality issue. The `convertToNumber` function is called twice for each comparison during sorting. This may affect performance, especially if the table contains a large number of rows. It would be more efficient to convert the salaries to numbers before sorting.